app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
```

Comment:
- [ ] Kindly design the login page to look identical to the wireframe provided for this project. 😎 

### Your Design

![image](https://github.com/otmanTR/BlogAppRoR/assets/77106962/0f98ded7-8f40-4bc4-ab68-f100e3bac738)

### Wireframe

![image](https://github.com/otmanTR/BlogAppRoR/assets/77106962/1d737514-6256-4969-b750-9d05220225a9)